### PR TITLE
I18n JS translations + system tests

### PR DESCRIPTION
**Pull request by lbiedinger**

### CODE REVIEW

**Comment by rwd** [comment]
@lbiedinger the system spec does not work for me, failing with:
```
  1) I18n-js Translations has I18n translations available in javascript
     Failure/Error: javascript_translated = evaluate_script("I18n.translate('site.browse.newcontent.description');")
     
     Capybara::Poltergeist::JavascriptError:
       One or more errors were raised in the Javascript code on the page. If you don't care about these errors, you can ignore them by setting js_errors: false in your Poltergeist configuration (see documentation for details).
     
       ReferenceError: Can't find variable: I18n
       ReferenceError: Can't find variable: I18n
           at undefined:1
           at :1
           at :1
     
     [Screenshot]: tmp/screenshots/failures_r_spec_example_groups_i18n_js_translations_has_i18n_translations_available_in_javascript_702.png


     
     # ./spec/system/i18n_js_translations_spec.rb:9:in `block (2 levels) in <top (required)>'
     # ./spec/spec_helper.rb:50:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:49:in `block (2 levels) in <top (required)>'
```

Do you have any idea why? Is there any per-environment configuration required before running RSpec?

File: README.md [hunk 1]
```diff
@@ -57,6 +57,10 @@ bundle exec rails s
 Now Europeana Stories will be accessible at http://localhost:3000/ and its
 admin interface at http://localhost:3000/admin
 
+## Testing
+
+System tests use the poltergeist gem which has an external dependency on PhantomJS. See here for installation instructions: http://phantomjs.org/documentation/.6.0#installing-phantomjs
```
Comment (rwd) [inline README.md]: That URL returns a 404

File: spec/capybara_helper.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Capybara.configure do |config|
+  config.default_max_wait_time = 10
+  config.default_selector = :css
+end
+
+if ENV['CAPYBARA_DRIVER'] == 'selenium'
+  require 'selenium-webdriver'
+
+  Capybara.server = :webrick
```
Comment (rwd) [inline spec/capybara_helper.rb]: Let's use Puma here. It's already added to the test group bundle by this PR, and that's what we use elsewhere so best to test against it.

File: spec/system/i18n_js_translations_spec.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+RSpec.describe 'I18n-js Translations' do
+
+  it 'has I18n translations available in javascript', type: :system, js: true do
```
Comment (rwd) [inline spec/system/i18n_js_translations_spec.rb]: `type: :system` is redundant here as it's inferred from the sub-directory of spec/

Comment (rwd) [reply spec/system/i18n_js_translations_spec.rb]: Except Travis seems to need it :( Will leave for now.

File: spec/capybara_helper.rb [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+# Capybara integration
+require 'capybara/rspec'
+require 'capybara/rails'
+
+Capybara.configure do |config|
+  config.default_max_wait_time = 10
+  config.default_selector = :css
+end
+
```
[the remark this answers is not in the document]
Comment (lbiedinger) [reply spec/capybara_helper.rb]: yes, I was going off this commit: https://github.com/teamcapybara/capybara/commit/1a07da3d10a2753415e79d1cdab5f6b078daef97
Which is going to switch the default to be puma, but isn't on any official release yet.

**Pull request merged**